miden/src/examples/fibonacci.rs

```diff
@@ -18,8 +18,8 @@ pub fn get_example(n: usize) -> Example {
 
     Example {
         program,
-        inputs: ProgramInputs::from_stack_inputs(&[0, 1]).unwrap(),
-        pub_inputs: vec![0, 1],
+        stack_inputs: StackInputs::try_from_values([0u64, 1]).unwrap(),
```

Comment:
minor nit: would prefer to write `0u64` as `0_u64`.

Reply:
Fixed. This was a leftover of the previous iteration that allowed arbitrary conversions to field elements